telemetry/DevHome.Telemetry/Telemetry.cs

```diff
@@ -184,9 +184,9 @@ public void LogTimeTaken(string eventName, uint timeTakenMilliseconds, Guid? rel
     /// <param name="eventName">The name of the event to log</param>

```

Comment:
It won't let me comment on line 182, but if you removed "informal" above, do you want to remove it here too?

Reply:
removed.